test/Tests/src/Semantic/Error_Spec.enso

```diff
@@ -66,7 +66,7 @@ spec =
 
         Test.specify "should implement to_text" <|
             Error.throw Nothing . to_text . should_equal "(Error: Nothing)"
-            Error.to_text . should_equal "Error"
+            Error.to_text Error . should_equal "Error"
```

Comment:
I get where this is coming from, but it is quite surprising. Should the `Error` argument be supported anymore?

Other than that - I'd like to see a test like
```
x = Error
x.to_text . should_equal "???"
```

What will this print? Will it be `Error` or a function now? Would be great to have a test documenting this behaviour, as it's non obvious.

Reply:
> What will this print? Will it be Error or a function now?
The latter. OK, I will add case.

Reply:
Hmm, is this definitely the way it is supposed to work? Shouldn't the _static_ approach only work for static references to the types?

Anyway, even if not, I'm not suggesting to do anything about it in the current PR.
Thanks for adding the test, documenting the currently expected behaviour.

Reply:
I would even argue this now works as it should have worked to be begin with. Let's take further discussion offline.